Tests/test_AlignIO.py

```diff
@@ -247,13 +248,21 @@ def check_summary_simple(self, alignment):
         dumb_consensus = summary.dumb_consensus()
         # gap_consensus = summary.gap_consensus()
 
-    def check_summary(self, alignment):
+    def check_summary(self, alignment, molecule_type):
         # Check AlignInfo.SummaryInfo likes the alignment; smoke test only
+        if molecule_type == "DNA":
+            letters = IUPACData.unambiguous_dna_letters
+        elif molecule_type == "RNA":
+            letters = IUPACData.unambiguous_rna_letters
+        elif molecule_type == "protein":
+            letters = IUPACData.protein_letters
+        else:
+            raise ValueError("Unknown molecule type '%s'" % molecule_type)
```

Comment:
What about mixed case? Or does this get passed though ``.upper()`` somewhere?

Reply:
Neither the original code nor the revised code consider this. This is a smoke test only.